shared/templates/template_BASH_regex_permissions

```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_all, Red Hat Enterprise Linux 5
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+find FILEPATH -regex '^FILEPATH/FILENAME' -exec chmod FILEMODE {} \;
```

Comment:
As far as I understand, the unix:filename operation="pattern match" is regex based, but maybe it is possible to use only globbing :-) In that case I can update the mechanism to use globbing only... and to simplify the chmod directly...
What is harder with globbing is to differentiate in the csv a globbing name (foo*) with a real file named foo\* which is not a globbing. Using ^ and $ is easier to detect regex (but regex specific). But maybe I consider complicated useless cases ... ;-)